src/lib/types/ImageCarousel.ts

```diff
@@ -0,0 +1,4 @@
+export interface ImageMetadata {
```

Comment:
Might as well use a bit of TypeScript 😁 `images` are of type `ImageMetadata` and the array consists of objects with `src` and `altText` properties

See https://github.com/lhollyer/the-small-society/pull/17/files#diff-864cd7499125efdabc67a4236aa1d65f51aa4848c8cc748e51b0a497aea550caR17-R58 for implementation